generated-platform-project/quarkus-qpid-jms/integration-tests/quarkus-qpid-jms-integration-tests/pom.xml

```diff
@@ -27,7 +27,7 @@
     <dependency>
       <groupId>io.quarkiverse.artemis</groupId>
       <artifactId>quarkus-test-artemis</artifactId>
-      <version>1.0.1</version>
+      <version>${quarkus-config-consul.version}</version>
```

Comment:
This is just an artifact. Apparently, it uses a property if only one property points to this particular version.

Reply:
Correct